storagenode/piecestore/endpoint_test.go

```diff
@@ -160,6 +160,88 @@ func TestUpload(t *testing.T) {
 	})
 }
 
+// TestSlowUpload tries to mock a SlowLoris attack.
+// The conditions to flag a slow loris attack:
+// +----------------------------------+-----------------------------------------------------------------------------------------------------------------------------------+
```

Comment:
This table wouldn't render nicely, it's cleaner to stick to regular language. e.g.

> Slow upload check will be triggered when upload speed falls below MinUploadSpeed and live concurrent connections is not over congestionThreshold.

And, this could be documentation on the check function itself, rather than the test itself.

Reply:
done